website/docs/d/cm_catalog.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "ibm"
+page_title: "IBM : cm_catalog"
+subcategory: "Catalog Management"
```

Comment:
this line should be above  `layout`.. something like
```
subcategory: "Catalog Management"
layout: "ibm"
```
please make this change in all other docs